tests/common.sh

```diff
@@ -140,7 +140,7 @@ function install_libraries()
     pushd $HOME/Arduino/libraries
 
     # install ArduinoJson library
-    { test -r ArduinoJson-v6.11.0.zip || wget -q -nv https://github.com/bblanchon/ArduinoJson/releases/download/v6.11.0/ArduinoJson-v6.11.0.zip; } && unzip -q ArduinoJson-v6.11.0.zip
+    { test -r ArduinoJson-v6.11.0.zip || curl --output ArduinoJson-v6.11.0.zip -L https://github.com/bblanchon/ArduinoJson/releases/download/v6.11.0/ArduinoJson-v6.11.0.zip; } && unzip -q ArduinoJson-v6.11.0.zip
```

Comment:
Should AruinoJson be a git submodule instead of being hardcoded?

Reply:
I would say not.  There is exactly 1 example we have w/JSON usage (not sure why it's there, but it's ancient).  All out submodules are 8266 specific, not generic Arduino-land (like this one).